k8s/postgresql/manifest/application.yaml.template

```diff
@@ -0,0 +1,59 @@
+    marketplace.cloud.google.com/deploy-info: '{partner_id: "google", product_id: "postgresql", partner_name: "Google Click To Deploy"}'
+  labels:
+    app.kubernetes.io/name: "$APP_INSTANCE_NAME"
+spec:
+  type: PostgreSQL
+  version: '9.6.10'
+  description: |-
+    PostgreSQL is an open source object-relational database with an emphasis on extensibility and standards-compliance.
+
+    # Support
+    Google does not offer support for this solution. However, community support is available on
+    [Stack Overflow](https://stackoverflow.com/questions/tagged/postgresql/).
+    Additional community support is available on [community forums](https://www.postgresql.org/list/).
+  info:
```

Comment:
Please remove info.

Reply:
instead of removing this section it would be better to deliver IP address of PostgreSQL instance

Reply:
info section can also deliver other important information, e.g. password to the database; as a reference, please, take a look on influxdb/manifest/application.yaml